docker-compose-clickhouse.yml

```diff
@@ -0,0 +1,52 @@
+---
+version: '3.7'
+
+services:
+  pmm-server-external-clickhouse:
+    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
+    container_name: pmm-server-clickhouse
+    depends_on:
+    - external-clickhouse
+    ports:
+      - "8080:80"
+      - "8443:443"
+    environment:
+      - PMM_DEBUG=1
+      - PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443
+      - PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
+      - PERCONA_TEST_VERSION_SERVICE_URL=https://check-dev.percona.com/versions/v1
+      - PERCONA_TEST_TELEMETRY_INTERVAL=10s
+      - PERCONA_TEST_TELEMETRY_RETRY_BACKOFF=10s
+      - PERCONA_TEST_PMM_CLICKHOUSE_ADDR=external-clickhouse:9000
+      - PERCONA_TEST_PMM_CLICKHOUSE_DATABASE=pmm
+      - PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE=1 
+      - PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE=65000
+  pmm-client-external-clickhouse:
+    image: ubuntu:latest
+    container_name: pmm-client
+    restart: on-failure
+    ports:
+      - "7778:7777"
+    command: >
+      bash -c ./home/config.sh
+    volumes:
+      - ./testdata/externalClickhouse/setupPMMClientUbuntu.sh:/home/config.sh
+
+  mysql5.7:
+    #Done
+    container_name: mysql5.7
+    image: mysql:5.7
+    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: pass
+    ports:
+      - "3306:3306"
```

Comment:
this host port will conflict with many other deployments, use something else, may be `3340:3306`